test/CMakeLists.txt

```diff
@@ -149,3 +149,9 @@ ADD_TEST( t_SensThickness_Clic_o2_v4 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${Pac
           ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_CURRENT_SOURCE_DIR}/../CLIC/compact/CLIC_o2_v04/CLIC_o2_v04.xml 300 50 )
 ADD_TEST( t_SensThickness_CLIC_o3_v15 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
           ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_CURRENT_SOURCE_DIR}/../CLIC/compact/CLIC_o3_v15/CLIC_o3_v15.xml 100 50 )
+
+#--------------------------------------------------
+# check if files named the same contain the same in FCCee
+SET( test_name "test_files_versions_FCCee" )
+ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh" "${CMAKE_SOURCE_DIR}/utils/compareIdenticalFiles.sh ${CMAKE_SOURCE_DIR}/FCCee ${CMAKE_SOURCE_DIR}/utils/IdenticalFiles_ToBeIgnored.txt" )
+SET_TESTS_PROPERTIES( t_${test_name} PROPERTIES FAIL_REGULAR_EXPRESSION  "Error. Files with the same name but different contents" )
```

Comment:
The test should fail itself if any file is different instead of parsing the output. Now, if you give a wrong input path for example the test will happily pass since it didn't even run and the string isn't there.

Reply:
done, thanks for the explanation!